flexmeasures/api/v3_0/sensors.py

```diff
@@ -273,6 +273,18 @@ def index(
     @route("<id>/data/upload", methods=["POST"])
     @path_and_files(SensorDataFileSchema)
     def upload_data(self, data: list[tb.BeliefsDataFrame], **kwargs):
+        """
```

Comment:
Please also add something so it is listed correctly in the API docs: Look at `post_data()`. Maybe: 

```
Post sensor data to FlexMeasures by file upload.

.. :quickref: Data; Upload sensor data by file
```